src/geo/projection/globe_util.js

```diff
@@ -428,6 +428,15 @@ function csLatLngToECEF(cosLat: number, sinLat: number, lng: number, radius: num
     return [sx, sy, sz];
 }
 
+export function ecefToLatLng(ecef: Array<number>): LngLat {
+    const radius = Math.sqrt(ecef[0] * ecef[0] + ecef[1] * ecef[1] + ecef[2] * ecef[2]);
+
+    const lng = Math.atan2(ecef[0], ecef[2]);
+    const lat = Math.PI * 0.5 - Math.acos(-ecef[1] / radius);
```

Comment:
Nit: much nicer with destructuring in cases like this.

```suggestion
export function ecefToLatLng([x, y, z]: Array<number>): LngLat {
    const radius = Math.hypot(x, y, z);
    const lng = Math.atan2(x, z);
    const lat = Math.PI * 0.5 - Math.acos(-y / radius);
```